marketdata/rest.go

```diff
@@ -1209,6 +1210,31 @@ func (c *client) GetLatestCryptoXBBO(symbol string, exchanges []string) (*Crypto
 	return &latestXBBOResp.XBBO, nil
 }
 
+// GetCryptoSnapshot returns the snapshot for a given crypto symbol
```

Comment:
[minor] If you already mention the symbol in the docstring, you should maybe also explain why the exchange is a required argument.